BlueprintUI/Sources/Measuring/SizeConstraint.swift

```diff
@@ -114,5 +127,51 @@ extension SizeConstraint {
             }
         }
 
+        /// Divides an Axis by a scalar value. If the Axis is unconstrained the
+        /// result will remain unconstrained.
+        public static func /(lhs: SizeConstraint.Axis, rhs: CGFloat) -> SizeConstraint.Axis {
```

Comment:
Think it's worth adding tests for these operators?

Reply:
Not a _ton_ of code in these, but it’s nice to have a seatbelt test just in case I mixed up a `+` and a `*` or something. Added.

Reply:
Thanks!